pkg/services/accesscontrol/roles.go

```diff
@@ -227,6 +227,7 @@ var FixedRoleGrants = map[string][]string{
 		usersOrgRead,
 	},
 	string(models.ROLE_ADMIN): {
+		datasourcesEditorRead,
```

Comment:
This is inherited from editors having that access already, do we want to duplicate the allocation?

Reply:
If it inherits then we don't need this 👍 I did not know that built-in role assignments inherit from each other, where is this being done? 🤔 Or am I misunderstanding the "inheriting" part?

Reply:
@vtorosyan It's done during evaluation, all the builtin roles are expanded to include all of their children. https://github.com/grafana/grafana/blob/f6019216706b29c744e77c3902eaca9dc4ec541a/pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go#L87-L97

Reply:
Thanks for the reference, I was not aware about this 👍 
I have remove the unnecessary changes and left only the note in the Explore doc to indicate that it can be used with fine-grained permissions.

Related, this behaviour seems important enough to document publicly, as the default assumption might be "Users have an access which I see when retrieving from the API".

Reply:
💯 agree!